code/modules/clothing/head/helmet.dm

```diff
@@ -243,6 +243,14 @@
 	item_state = "skull"
 	strip_delay = 100
 
+/obj/item/clothing/head/helmet/durathread
+	name = "makeshift helmet"
+	desc = "A hardhat with strips of leather and durathread for additional blunt protection."
+	icon_state = "durathread"
+	item_state = "durathread"
+	armor = list("melee" = 25, "bullet" = 10, "laser" = 20,"energy" = 10, "bomb" = 30, "bio" = 15, "rad" = 20, "fire" = 100, "acid" = 50)

```

Comment:
For comparison, this is **_more powerful_** than a skull helmet made out of just bones versus fire, indirect bomb damage & acid while the skull helmet compensates for more rounded statistics versus more conventional damage types.

- More hardhats can be found in the 'engineering supplies' cargo crate to dismiss limited supply reasoning & concerns for crafting, at up to 3 hats a crate.

Reply:
those stats are also from the hard hat themselves so